packages/ui/lib/NoData/index.js

```diff
@@ -1,18 +1,18 @@
 import React from "react";
 import styled from "styled-components";
 import FlexCenter from "../styled/FlexCenter";
-import { p_14_normal } from "../styles/textStyles";
+import Card from "../Card";
+import { text_dark_accessory } from "../styles/colors";
 
-const Wrapper = styled(FlexCenter)`
-  background: #ffffff;
-  height: 131px;
-  ${p_14_normal};
-  color: #9da9bb;
-  border: 1px solid #f0f3f8;
-  box-shadow: 0px 4px 31px rgba(26, 33, 44, 0.04),
-    0px 0.751293px 3.88168px rgba(26, 33, 44, 0.03);
+const Message = styled(FlexCenter)`
+  color: ${text_dark_accessory};
+  height: 100%;
```

Comment:
can we define a `height: 130px` here and not using a inline-css below?

Reply:
it is because the `card` component has default padding(32px), the `nodata` design draft height is 130px,

if so, line 9 height should be `height: 66px` + card's padding `32px * 2` = `130px`

Reply:
66px is fine